src/mapml/handlers/ContextMenu.js

```diff
@@ -697,5 +700,38 @@ export var ContextMenu = L.Handler.extend({
   _onItemMouseOut: function (e) {
     L.DomUtil.removeClass(e.target || e.srcElement, 'over');
     this._hideCoordMenu(e);
+  },
+
+  toggleContextMenuItems: function (options,state) {
+    if (state === "disabled"){
+      if (options === "Toggle Controls"){
```

Comment:
See previous comments about using a case-insensitive single token, not a full string.